pkg/variantregistry/ocp.go

```diff
@@ -407,6 +410,17 @@ func (v *OCPVariantLoader) IdentifyVariants(jLog logrus.FieldLogger, jobName str
 		variants[VariantSuite] = "unknown" // parallel perhaps but lots of jobs aren't running out suites
 	}
 
+	if etcdScaling.MatchString(jobName) {
+		variants[VariantProcedure] = "etcd-scaling"
+		variants[VariantRarelyRun] = "true"
+	} else if cpuPartitioning.MatchString(jobName) {
+		variants[VariantProcedure] = "cpu-partitioning"
+		variants[VariantRarelyRun] = "true"
```

Comment:
Could we find a way to avoid a boolean?

What about putting jobs into tier buckets? Experimental jobs might be automatically excluded from component readiness, and it would let us remove the SQL hacks that filter out specific jobs. Maybe call it JobTier?

| **Category**    | **Description**                                      | **Default Analysis**          |
|-----------------|------------------------------------------------------|------------------------|
| `rare`          | Runs less than daily, used for rarely run analysis   | Rarely run    |
| `standard`      | Runs at least daily, not on payloads                 | Fishers, pass rate             |
| `informing`     | Informing on payloads                                | Fishers, pass rate               |
| `blocking`      | Blocking on payloads                                 | Fishers, pass rate               |
| `experimental`  | Default ignored in component readiness (CR)          | None                |

Reply:
The data we should already have, too. For informing/blocking, Ken added that data in the Sippy config a while back: https://github.com/openshift/sippy/blob/master/config/openshift.yaml#L8409

Reply:
I like this, I'll at least add the variant and rare for now and we can expand with the others in a followup. may get it all today, will see.

Reply:
Pushed!

Reply:
Fantastic idea.